### PR TITLE
Update hot-reload.md

### LINKED ISSUE #25132: .Net Hot Reload not working
I have Visual Studio 16.11.3 and a .NET 5.0 Application. The Hot Load does not appear to be working. 

What are the minimum requirements for .NET Hot Reload?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 41ab3e64-1abc-987b-13db-4e49f2d68ff3
* Version Independent ID: 41ab3e64-1abc-987b-13db-4e49f2d68ff3
* Content: [.NET Hot Reload support for ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/test/hot-reload?view=aspnetcore-6.0)
* Content Source: [aspnetcore/test/hot-reload.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/test/hot-reload.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-test**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #25132

### CODE REVIEW

**Comment by guardrex** [comment]
I defer to Artak. I'm not familiar enough with the tech to know if the cross-links contain valid content.

File: aspnetcore/test/hot-reload.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ dotnet watch --no-hot-reload
 
 For more information, see the following resources in the Visual Studio documentation:
 
-* [Visual Studio 2022 version 17.0 RC and Preview Release Notes: .NET Hot Reload](/visualstudio/releases/2022/release-notes-preview#net-hot-reload)
+* YouTube video [.NET 6 Hot Reload in Visual Studio 2022, VS Code, and NOTEPAD?!?](https://www.youtube.com/watch?v=4S3vPzawnoQ)
+* [Introducing the .NET Hot Reload experience for editing code at runtime](https://devblogs.microsoft.com/dotnet/introducing-net-hot-reload/)
+* <!--I can't find hot in that link.[Visual Studio 2022 version 17.0 RC and Preview Release Notes: .NET Hot Reload](/visualstudio/releases/2022/release-notes-preview#net-hot-reload) -->
```
Comment (Rick-Anderson) [inline aspnetcore/test/hot-reload.md:68]:
```suggestion
* [Write and debug running code with Hot Reload in Visual Studio](/visualstudio/debugger/hot-reload)
*  <!--I can't find hot in that link.[Visual Studio 2022 version 17.0 RC and Preview Release Notes: .NET Hot Reload](/visualstudio/releases/2022/release-notes-preview#net-hot-reload) -->
```

**Comment by mkArtakMSFT** [comment]
Thanks for emailing me, @guardrex . I'll review this in an hour or so, @Rick-Anderson !

**Review by mkArtakMSFT — APPROVED** [review]
I like the idea of linking to other resources that we publish! Thanks @Rick-Anderson !

**Pull request merged**